test/one_hot_encoding.jl

```diff
@@ -35,9 +35,6 @@
             @test FeatureTransforms.apply(x, ohe; inds=2:4) == [0 0 1; 0 1 0; 0 0 1]
             @test FeatureTransforms.apply(x, ohe; dims=:) == expected
 
-            @test_throws DimensionMismatch FeatureTransforms.apply(x, ohe; dims=1)
```

Comment:
Why are these removed?

Reply:
had to do with removing `mapslices` - `selectdim` doesn't through a `DimensionMismatch`. It'll be in another PR